draft-ietf-trans-rfc6962-bis.md

```diff
@@ -265,8 +265,8 @@ algorithms, see {{hash_algorithms}}. The hashing algorithm in use is referred to
 as HASH throughout this document and the size of its output in bytes as
 HASH_SIZE. The input to the Merkle Tree Hash is a list of data entries; these
 entries will be hashed to form the leaves of the Merkle Hash Tree. The output is
-a single HASH_SIZE Merkle Tree Hash. Given an ordered list of n inputs, D\[n] =
-{d(0), d(1), ..., d(n-1)}, the Merkle Tree Hash (MTH) is thus defined as
+a single HASH_SIZE Merkle Tree Hash. Given an ordered list of n inputs, D^(n) =
```

Comment:
Mostly. We struggled with getting the notation for this right from the beginning. Not convinced this is less confusing, but I don't think it is _more_ confusing.

The one other change I might make I will suggest in a moment. :-)